react/props.md

```diff
@@ -127,9 +167,11 @@ const Greeter = (props) => {
 export default Greeter
 ```
 
+<img src="./assets/props-pipeline.png" alt="basic visual representation of props pipeline between App.js and Greeter.js" height="40%">
+
 ### Passing Values From State as Props
 
-Rather than hard coding the string "Hamilton" directly in the component call we can use a state value to pass props to the nested component. This will make our application more dynamic as state values can be updated.
+Now that we are using props in our greeter application, we can being to refactor. Our refactor can improve the code in our application. Rather than hardcoding the string "Hamilton" directly in the component call we can use a state value to pass props to the Greeter component. This will make our application more dynamic as state values can be updated.
```

Comment:
```suggestion
Now that we are using props in our greeter application, we can begin to refactor. Our refactor can improve the code in our application. Rather than hardcoding the string "Hamilton" directly in the component call we can use a state value to pass props to the Greeter component. This will make our application more dynamic as state values can be updated.
```